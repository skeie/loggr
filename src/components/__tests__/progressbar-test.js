import React from 'react';
import renderer from 'react-test-renderer';
import Progressbar from '../Progressbar';
test('Should render Progressbar correctly', () => {
    const tree = renderer.create(
        <Progressbar /> 
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
    