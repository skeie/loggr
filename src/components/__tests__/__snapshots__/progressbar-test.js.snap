exports[`test Should render Progressbar correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small" />
</View>
`;
